osbs/build/config_map_response.py

```diff
@@ -34,10 +35,23 @@ def get_data(self):
         """
         Find the data stored in the config_map
 
-        :return: dict, the json data that was passed into the ConfigMap on creation
+        :return: dict, the data that was passed into the ConfigMap on creation
         """
         data = graceful_chain_get(self.json, "data")
         if data is None:
             return {}
 
         return data
+
+    def get_json(self):
+        """
+        Find the data stored in the config_map
+
+        :return: json, the json version of the json data that was passed into the ConfigMap on
+                 creation
+        """
+        data = graceful_chain_get(self.json, "data")
+        if data is None:
+            return {}
+
+        return json.dumps(data)
```

Comment:
do I test that by doing
assert config_map.get_json("special.how") == "very"
or something else?